From 7cb87e4070ac320e9b990bb90b19aa69b032b9d3 Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Sun, 26 Apr 2020 22:30:54 +0300
Subject: [PATCH] [SW-0] goya2: Disable fdt system fixup for onix

Disable fdt system fixup for onix as peripherals
are not connected on onix and fixups are giving errors
in logs.
Disabling CONFIG_OF_SYSTEM_SETUP in goya2 Kconfig and
enabling it in asic/pldm fragments.

Note: Disabling CONFIG_OF_SYSTEM_SETUP=n is uboot_onix.fragment
is not working

Change-Id: Ib6ec3a6c8774008cd8873e8e8e645d41971aa076
---

diff --git a/board/habanalabs/goya2/uboot_asic.fragment b/board/habanalabs/goya2/uboot_asic.fragment
index 7151225..3f87dd7 100644
--- a/board/habanalabs/goya2/uboot_asic.fragment
+++ b/board/habanalabs/goya2/uboot_asic.fragment
@@ -9,3 +9,4 @@
 CONFIG_SYS_EEPROM_PAGE_WRITE_DELAY_MS=5
 CONFIG_SYS_I2C_EEPROM_ADDR=0x54
 CONFIG_SYS_I2C_EEPROM_ADDR_LEN=1
+CONFIG_OF_SYSTEM_SETUP=y
diff --git a/board/habanalabs/goya2/uboot_pldm.fragment b/board/habanalabs/goya2/uboot_pldm.fragment
index ee37ece..a5a7378 100644
--- a/board/habanalabs/goya2/uboot_pldm.fragment
+++ b/board/habanalabs/goya2/uboot_pldm.fragment
@@ -6,3 +6,4 @@
 CONFIG_DEFAULT_DEVICE_TREE="goya2_pldm"
 CONFIG_PHY_FIXED=y
 CONFIG_PLDM=y
+CONFIG_OF_SYSTEM_SETUP=y
